Kitodo/src/main/resources/messages/messages_de.properties

```diff
@@ -571,6 +571,7 @@ importConfig.searchField=Suchfeld
 importConfig.searchFields=Suchfelder
 importConfig.searchFields.defaultField=Default-Suchfeld
 importConfig.searchFields.idSearchField=ID-Parameter
+importConfig.searchFields.idPrefix=ID-Prefix
```

Comment:
"Prefix" -> "Präfix" ... looks like you used a lot of English words one to one in German like "Default", "ID-Parameter" and maybe more.

Reply:
Thanks for pointing out the "ID-Prefix", I will change this. But what's wront about "ID-Parameter"? Parameter is a german word, too, I don't see the problem here. English terms like "Default" are also very establised in the German language, so I thought it would be appropriate here. Or would you prefer something like "Standardsuchfeld"?

Reply:
ID Parameter looks for me wrong but I did not have for now a better wording. Maybe ask a librarian or other real user which wording is better.